opal/mca/btl/tcp/btl_tcp_endpoint.c

```diff
@@ -848,7 +848,7 @@ static int mca_btl_tcp_endpoint_complete_connect(mca_btl_base_endpoint_t* btl_en
         opal_show_help("help-mpi-btl-tcp.txt", "client connect fail",
                        true, opal_process_info.nodename,
                        getpid(), msg,
-                       strerror(opal_socket_errno), opal_socket_errno);
+                       strerror(so_erro), so_erro);
```

Comment:
Don't you need to call `getsockopt()` to read the `SO_ERROR` value?

Reply:
Yes, that's what we do about 20 lines above. The CI error was due to a typo (because I failed my copy/paste from another terminal). Should be fixed now.